src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -41,6 +68,40 @@ function ScrollDown(props) {
   );
 }
 
+function RetrieveStats() {
+  const { telescopeUrl } = useSiteMetadata();
+  const [stats, setStats] = useState({ stats: { posts: 0, authors: 0, words: 0 } });
+
+  useEffect(() => {
+    async function getStats() {
+      try {
+        const response = await fetch(`${telescopeUrl}/stats/year`);
+        if (response.status !== 200) {
+          throw new Error(response.statusText);
+        }
+
+        const stat = await response.json();
+        const locale_stats = {
```

Comment:
ESLint didn't like that `locale_stats` wasn't in camel case

Reply:
I've been writing too much Python recently :face_with_head_bandage:, thanks for pointing out!